docker/scripts/install-packages.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -e
+
+echo '===================='
+echo '- Install packages -'
+echo '===================='
+
+# Unset Connect URL to prevent nuxeoctl from reaching Connect:
+# - We rely on local packages.
+# - The network might not be available.
+# - The Connect server might not be responding.
+connectUrlProp=org.nuxeo.connect.url=
+echo $connectUrlProp >> $NUXEO_CONF \
+
+# List and install packages
+packagesDir=/packages
+packages=$(find $packagesDir -name *.zip)
+if [ ! -z "$packages" ]; then
+  echo 'Packages to install:'
+  cat << EOF
+$packages
+EOF
+  echo
+  $NUXEO_HOME/bin/nuxeoctl mp-install --accept yes --nodeps $packages
+else
+  echo 'Found no packages to install.'
+fi
+echo
+
+# Reset Connect URL
+sed -i "/$connectUrlProp/d" $NUXEO_CONF
```

Comment:
Shouldn't we add `^` and `$` to prevent removal of the property declared by user ?

Reply:
In fact, the script is executed at build time on the raw nuxeo.conf from the distribution, so it should be safe.
Besides, the line is not necessarily at the end of the file since the `mp-install` command run in the meantime has added some properties.